programmers/문자열-다루기-기본.js

```diff
@@ -0,0 +1,15 @@
+function solution(s) {
+    
+    let str = s.toUpperCase();
+    if (str === s) {
+        if (str.toLowerCase() === s) {
+            if (str.length === 4 || str.length === 6) {
+                return true;
+            }
+            return false;
+        }
+        return false;
+    } else {
+     return false;   
+    }
+}
```

Comment:
```suggestion
function solution(s) {
    const str = s.toUpperCase();
    
    if (str === s && (str.length === 4 || str.length === 6)) {
        return true;
    }
    return false;
}
```
toLowerCase가 없어도 문제가 풀리긴하는데,  이런경우까지 생각해주는거 너무좋아여
조건문에서 다음 조건문을 합쳐서 if문의 갯수랑 코드길이를 줄일 수 있어요

```js
function solution(s) {
    const temp = Number(s)
    
    return (s.length === 4 || s.length === 6) && temp === parseInt(s) ? true : false;
};
```
이런 방법이 있습니다 ~
parseInt와 Number 차이에 대해서 공부해오세요😀